<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <arg name="base_link" default="base_link"/>
    <arg name="base_footprint" default="base_footprint"/>
    <arg name="z_offset" default="0.0"/>

    <node name="base_footprint_pub" pkg="tf2_ros" type="static_transform_publisher"
          args="0. 0. $(arg z_offset) 0. 0. 0. 1. $(arg base_link) $(arg base_footprint)" output="log"/>
</launch>
